Getting-Started/Design/Rendering-Media/index.md

```diff
@@ -1,40 +1,71 @@
 # Rendering media
 
-_Templates can access items in the Media library, to assist in displaying rich content like galleries_
+_Templates can access items in the [Media library](../../Data/Creating-Media/index.md), to assist in displaying rich content like galleries_
 
-## Rendering a single media item
-A media node is not just a file, but a collection of fields, such width, height and the path to the stored file.
+In the following examples we will be looking at rendering out an Image, however the core principles do apply to all media items but property aliases may differ.
 
+## Rendering a media item
+A media node is not just a file, but like content, it is a collection of fields, such width, height and the path to the stored file. The benefit of this is that accessing media is very similar to accessing a content node.
+
+### Example 1: Accessing an image media item based on its ID
 A standard image in the media library is based on the Mediatype `image` which provides a number of standard values - if you want to add more, simply edit the media type under **settings**.
 
+_Assumption: We are going to assume that our media item has an ID of **1234**, and that we are **not using Models Builder**_
+
     @{
-        var mediaItem = Umbraco.Media(1234);
-        var height = mediaItem.UmbracoHeight;
-        var file = mediaItem.UmbracoFile;
+        var mediaItem = Umbraco.TypedMedia(1234);
+        var height = mediaItem.GetPropertyValue<string>("umbracoHeight");
+
+        //There are two ways to get your media item url, this is one way.
+        var file = mediaItem.GetPropertyValue<string>("umbracoFile");
+
+        //This is the other
+        var url = mediaItem.Url
     }
 
     <img src="@file" height="@height" />
 
-## Rendering media item with Image Cropper
-If the Image Cropper is used as the upload field (umbracoFile) on a media type you can get the cropped image by calling the `GetCropUrl` function and referencing a crop alias.
+But wait a second, why do we have to access properties in the old IPublishedContent approach when Umbraco now comes with [ModelsBuilder](../../../Reference/Templating/Modelsbuilder/index.md) you may be asking yourself. Well, if you are using Umbraco version 7.4.0 or newer, you don't. You can happily use a nicely TypedModel for your media item if ModelsBuilder is enabled.
```

Comment:
IPublishedContent is not a old approach, you are referring to the the "stringly typed" approach, I'm not sure what we should call it officially though.... Perhaps just focus on the ModelsBuilder approach...

Reply:
I agree the wording isn't quite right looking back on it. But I also think we should keep both examples in. Not everyone is using ModelsBuilder, so having both GetPropertValue examples and Strongly typed examples would help. What about if we re-worded it to the classic/traditional approach when referring to GetPropertyValue?

Reply:
@NikRimington absolutely we should have both, I wasn't suggesting removing the "stringly" typed examples I was suggesting about the wording of this paragraph to focus on Models Builder,
Something like:
------------------------------------------------------------------
If you are using the ModelsBuilder included with Umbraco v7.4.0+ you use the strongly types models for your media items
------------------------------------------------------------------